packages/apps/graph/docker-compose.yaml

```diff
@@ -1,6 +1,6 @@
 services:
   graphql-source:
-    image: kadena/devnet
+    image: ghcr.io/kadena-community/graph-source
```

Comment:
They should both be `graph`. The tags make the difference (ie, the build from source is `preview` and `dev`, where the one from the package on npm is `latest`)